drivers/entropy/entropy_stm32.c

```diff
@@ -140,12 +140,15 @@ static int entropy_stm32_suspend(void)
 	LL_RNG_SetAesReset(rng, 1);
 #endif /* CONFIG_SOC_STM32WB09XX */
 
-#ifdef CONFIG_SOC_SERIES_STM32WBAX
-	uint32_t wait_cycles, rng_rate;
-
-	if (LL_PKA_IsEnabled(PKA)) {
+#if defined(PKA)
```

Comment:
Please add a comment to explicit that we're talking about the use of PKA "outside of zephyr" as not supported yet.
(otherwise this wouldn't be implemented this way).